### PR TITLE
Replace fabricbot with GHA for scheduled issue closing

### PR DESCRIPTION
**Pull request by jakebailey**
fabricbot is no more; this PR replaces all of the scheduled jobs with a GHA workflow that will close the issues. (Nicely, this will also close the issues with the correct "reason".)

There are other things to replace out of `fabricbot.json` but they'll require a different workflow.

### CODE REVIEW

File: .github/workflows/close-issues.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+          close_issues() {
+            echo "Closing issues marked as '$1'."
+            for issue in $(gh issue list --label "$1" --repo ${{ github.repository }} --state open --search "updated:<$DATE" --json number --jq '.[].number'); do
+              echo "Closing https://github.com/${{ github.repository }}/issues/$issue"
+              gh issue close $issue --reason "not planned" --comment "This issue has been marked as '$1' and has seen no recent activity. It has been automatically closed for house-keeping purposes."
+            done
+          }
+
+          close_issues "Duplicate"
```
Comment (RyanCavanaugh) [inline .github/workflows/close-issues.yml:38]:
```suggestion
          close_issues "Duplicate"
          close_issues "Unactionable"
          close_issues "Not a Defect"
```

Comment (jakebailey) [reply .github/workflows/close-issues.yml:38]: I had been meaning to ask about those; we don't auto-close them but feels like we should. I'll tack it on.

Comment (jakebailey) [reply .github/workflows/close-issues.yml:38]: There are some 80 issues labeled with "Not a Defect" that are still open; hopefully it won't get rate limited.

**Review by weswigham — APPROVED** [review]

**Comment by jakebailey** [comment]
The smoke/package-size failures are not related and I'm trying to diagnose them separately; I'm going to merge this now so I can verify that it is going to work.

**Pull request merged**